engine.py

```diff
@@ -64,7 +60,11 @@ def toolkit_manager(self):
                   in a separate external process (for example when you launch an app
                   such as the publisher from Shotgun Create).
         """
-        return self._toolkit_manager
+        try:
+            from sgtk.platform.qt import QtCore
+            return QtCore.QCoreApplication.instance().findChild(QtCore.QObject, "sgtk-manager")
+        except:
```

Comment:
`except Exception`

You'll get the result you're looking for without also trapping system-level exceptions, like keyboard interrupts, etc.